docs/04.guides/04.cookbooks/29.Threads-Scope/page.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Threads
+id: thread_usage
+---
+## Thread Scope ##
+
+This document explains how to use the threads. Threads mainly used for execute the code parallel.
+
+### Example 1 ###
+
+This below example shows waiting normal execution waiting for all code to resolve. Here it waits for 1000 mili seconds after the it complete the execution.
+
+```lucee
+<cfscript>
+function napASecond() localmode=true {
+	sleep(1000);
+}
+start=getTickCount();
+napASecond();
+dump("done in #getTickCount()-start#ms");
+</cfscript>
+```
+
+This below example shows thread execution. Code with in the thread tag run parallel to execution. Here execution doesn't wait for sleep.
```

Comment:
***ToChange***
The below example shows thread execution. Code within the thread tag runs in parallel with the execution of the other code in the cfscript. Here execution does not wait for sleep.